content/ai/concepts/search-algorithms/terms/breadth-first-search/breadth-first-search.md

```diff
@@ -0,0 +1,117 @@
+---
+Title: 'Breadth-First Search'
+Description: 'A shortest-path and traversing algorithm for unweighted graphs.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Graphs'
+  - 'Graph Search'
+  - 'Algorithms'
+CatalogContent:
+  - 'learn-python'
+  - 'paths/computer-science'
+---
+
+**Breadth-first search (BFS)** is a traversing algorithm for unweighted graphs. This is a foundational algorithm in [graph](https://www.codecademy.com/resources/docs/general/graph) theory from which many other algorithms start.
+
+## Features
+
+Some of the features and constraints that define the use and functionality of a breadth-first search algorithm include the following:
+
+- A relatively simple and efficient algorithm for finding the shortest-path in graphs that do not have edge weights.
+- It has a time complexity of _O(|V| + |E|)_, which is the sum of the vertices and edges.
```

Comment:
@caupolicandiaz Can we put it in bold instead?
```suggestion
- It has a time complexity of **O(|V| + |E|)**, which is the sum of the vertices and edges.
```

Reply:
all set